### PR TITLE
<fill-mask>Modify codes so that different accelerators can be called according to specific device conditions

### PR DESCRIPTION
**Pull request by foin6**
Modifications in: DeepSpeedExamples/inference/huggingface/fill-mask :

1. use "deepspeed.accelerator.get_accelerator()" to replace "cuda" specific codes；

p.s. Install intel-extension-for-deepspeed and intel-extension-for-pytorch before running.

### CODE REVIEW

**Review by rogerxfeng8 — COMMENTED** [review]
Looks good to me

File: inference/huggingface/fill-mask/test-bert.py [hunk 1]
```diff
@@ -4,13 +4,14 @@
 import torch
 import os
 import argparse
+from deepspeed.accelerator import get_accelerator
 
 parser = argparse.ArgumentParser()
 parser.add_argument("--model", "-m", type=str, help="hf model name")
-parser.add_argument("--dtype", type=str, default="fp16", help="fp16 or fp32")
```
Comment (delock) [inline inference/huggingface/fill-mask/test-bert.py:10]: Keep naming convention consistency: bfloat16 --> bf16.

Comment (foin6) [reply inference/huggingface/fill-mask/test-bert.py:10]: Solved

**Comment by delock** [comment]
looks good for me
